# Exception tests.

test_check_success(AssertNoExcept)
test_check_success(NoExceptPredicate)
